libs/auth-api-lib/src/lib/entities/models/personal-representative-scope-permission.model.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  Column,
+  DataType,
+  CreatedAt,
+  UpdatedAt,
+  Model,
+  Table,
+  PrimaryKey,
+  ForeignKey,
+} from 'sequelize-typescript'
+import { ApiProperty } from '@nestjs/swagger'
+import { PersonalRepresentativeRightType } from './personal-representative-right-type.model'
+import { ApiScope } from './api-scope.model'
+
+@Table({
+  tableName: 'personal_representative_scope_permission',
+})
```

Comment:
Note that some other join tables in this project use composite primary keys instead of a separate uuid key. I don't mind either way, but you might want to add the unique constraint with some Sequelize decorator so the test database is in sync with the migrated database.